### PR TITLE
[10.0][IMP] mass_editing: Take into account last changes

### PR DESCRIPTION
**Pull request by rousseldenis**
Depends on:

* [x] https://github.com/OCA/server-tools/pull/1856

### CODE REVIEW

File: attribute_set_mass_edit/wizards/mass_editing_wizard.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+        res = super(MassEditingWizard, self)._get_field_options(field)
+        if field.ttype in ("many2one", "many2many"):
+            attributes = self.env["attribute.attribute"].search(
+                [("field_id", "=", field.id)]
+            )
+            attribute = first(
+                attributes.filtered(
+                    lambda a: a.attribute_type in ("select", "multiselect")
+                )
+            )
```
Comment (simahawk) [inline attribute_set_mass_edit/wizards/mass_editing_wizard.py:22]: @rousseldenis can't we do simply this?
```suggestion
            attribute = self.env["attribute.attribute"].search([
                ("field_id", "=", field.id),
                ("attribute_type", "in", ("select", "multiselect"))
            ], limit=1)
```

File: test-requirements.txt [hunk 2]
```diff
@@ -1,4 +1,4 @@
 --extra-index-url=https://wheelhouse.shopinvader.com/simple
 odoo_test_helper
-odoo10-addon-mass-editing
 odoo10-addon-base_search_custom_field_filter @ git+https://github.com/OCA/server-ux@refs/pull/170/head#subdirectory=setup/base_search_custom_field_filter
+odoo10-addon-mass-editing @ git+https://github.com/OCA/server-tools@refs/pull/1856/head#subdirectory=setup/mass_editing
```
Comment (simahawk) [inline test-requirements.txt]: this is merging now :wink:

**Comment by lmignon** [comment]
@rousseldenis can you update the dependencies since  odoo10-addon-mass-editing is merged.

**Comment by rousseldenis** [comment]
Done

**Review by lmignon — APPROVED** [review]

**Pull request closed without merging**